auth/shared-signal-receiver/tests/unit/handlers/credential-change-handler.unit.test.ts

```diff
@@ -37,16 +48,134 @@ describe('handleCredentialChangeRequest', () => {
     vi.clearAllMocks();
     consoleInfoMock.mockReset();
     consoleErrorMock.mockReset();
+    consoleWarnMock.mockReset();
   });
 
   afterAll(() => {
     consoleInfoMock.mockRestore();
     consoleErrorMock.mockRestore();
+    consoleWarnMock.mockRestore();
+  });
+
+  it('return ACCEPTED when User not found for password change', async () => {
+    verifyUserExistsMock.mockResolvedValue(false);
+
+    const input = {
+      iss: 'https://identity.example.com',
+      jti: '123e4567-e89b-12d3-a456-426614174000',
+      iat: 1721126400,
+      aud: 'https://service.example.gov.uk',
+      events: {
+        'https://schemas.openid.net/secevent/caep/event-type/credential-change':
+          {
+            change_type: 'update',
+            credential_type: 'password',
+            subject: {
+              uri: 'urn:example:account:1234567890',
+              format: 'urn:example:format:account-id',
+            },
+          },
+        'https://vocab.account.gov.uk/secevent/v1/credentialChange/eventInformation':
+          {
+            email: 'any@test.com',
+          },
+      },
+    } as CredentialChangeRequest;
+
+    const response = await handleCredentialChangeRequest(input);
+
+    expect(consoleInfoMock).toHaveBeenCalledWith(
+      'CorrelationId: ',
+      '123e4567-e89b-12d3-a456-426614174000',
+    );
+
+    expect(verifyUserExists as Mock).toHaveBeenCalledWith(
+      'urn:example:account:1234567890',
+    );
+
+    expect(consoleWarnMock).toHaveBeenCalledWith('SIGNAL_WARN_USER_NOT_FOUND', {
+      userId: 'urn:example:account:1234567890',
+      correlationId: '123e4567-e89b-12d3-a456-426614174000',
+      requestType: 'UPDATE_PASSWORD',
+    });
+
+    expect(adminGlobalSignOutMock).not.toHaveBeenCalled();
+
+    expect(response).toEqual({
+      body: JSON.stringify({
+        message: ReasonPhrases.ACCEPTED,
+      }),
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      statusCode: StatusCodes.ACCEPTED,
+    });
+
+    verifyUserExistsMock.mockReset();
+  });
+
+  it('return ACCEPTED when User not found for email change', async () => {
+    verifyUserExistsMock.mockResolvedValue(false);
+
+    const adminGlobalSignOutMock = adminGlobalSignOut as Mock;
+
+    const input = {
+      iss: 'https://identity.example.com',
+      jti: '123e4567-e89b-12d3-a456-426614174000',
+      iat: 1721126400,
+      aud: 'https://service.example.gov.uk',
+      events: {
+        'https://schemas.openid.net/secevent/caep/event-type/credential-change':
+          {
+            change_type: 'update',
+            credential_type: 'email',
+            subject: {
+              uri: 'urn:example:account:1234567890',
+              format: 'urn:example:format:account-id',
+            },
+          },
+        'https://vocab.account.gov.uk/secevent/v1/credentialChange/eventInformation':
+          {
+            email: 'any@test.com',
+          },
+      },
+    } as CredentialChangeRequest;
```

Comment:
If this is the same data across multiple tests (with maybe one or two fields different) can we create a helper function to generate this payload?